README.md

```diff
@@ -325,6 +326,13 @@ be erased from 0x40000 to 0x48000,  but address 0x40000 will have value
 Act as a proxy for the host serial console when it is jumpered to the SP.
 
 
+### `humility counters`
+
+`humility counters` reads and displays any Hubris counters (as created via
+the `counters` crate and either the `counters!` or `counted_ringbuf!`
+macros).
+
+
```

Comment:
Do you need to re-run `cargo xtask readme`?  There are examples in the crate docstring which don't show up here.

Reply:
gah, whoopsie!